config/ompi_check_psm2.m4

```diff
@@ -40,6 +40,10 @@ AC_DEFUN([OMPI_CHECK_PSM2],[
 				    [Search for PSM (Intel PSM2) libraries in DIR])])
 	OPAL_CHECK_WITHDIR([psm2-libdir], [$with_psm2_libdir], [libpsm2.*])
 
+        AC_ARG_ENABLE([psm2-version-check],
+                  [AC_HELP_STRING([--enable-psm2-version-check],
```

Comment:
Would prefer '--disable-psm2-version-check' and default be 'no' since it implies that disabling the check is an exception, not the norm.

Of course semantically it makes no difference.